test/run_test

```diff
@@ -34,6 +35,8 @@ test -n "${OS-}" || false 'make sure $OS is defined'
 CIDFILE_DIR=$(mktemp --suffix=postgresql_test_cidfiles -d)
 
 volumes_to_clean=
+s2i_args="--pull-policy=never "
+test_dir="$(readlink -zf $(dirname "${BASH_SOURCE[0]}"))"
```

Comment:
`test_dir=$(readlink -f "$(dirname "$0")")` could work too, preferably should be moved to `run_s2i_test` too as a local variable.